src/Col.js

```diff
@@ -46,20 +47,25 @@ const Col = (props) => {
   const {
     className,
     cssModule,
+    widths,
     ...attributes
   } = props;
   const colClasses = [];
 
-  colWidths.forEach(colWidth => {
-    const columnProp = props[colWidth];
+  widths.forEach((colWidth, i) => {
+    let columnProp = props[colWidth];
+
+    if (!i && columnProp === undefined) {
+      columnProp = true;
+    }
 
     delete attributes[colWidth];
 
     if (!columnProp) {
       return;
     }
 
-    const isXs = colWidth === 'xs';
+    const isXs = !i;
```

Comment:
Yeah, but I probably should have just went with `i === 0` or something to be more clear. That's my bad.